sqs/user_template.go

```diff
@@ -0,0 +1,147 @@
+package sqs
+
+import (
+	"fmt"
+
+	goformation "github.com/awslabs/goformation/v4/cloudformation"
+	goformationiam "github.com/awslabs/goformation/v4/cloudformation/iam"
+	goformationtags "github.com/awslabs/goformation/v4/cloudformation/tags"
+)
+
+const (
+	SQSResourceIAMUserResourceName      = "IAMUser"
```

Comment:
we should maybe tidy up the SQS references in this file

Reply:
well that was a mission ... I probably shoulda let that slide ... I've renamed all the constants to be much more clear and consist.

all resource constants are now `ResourceXXXX` 

all output constants are now `OutputXXX`

"DL" is replaced with Deadletter to make it less ambiguous

names are simplified